README.md

```diff
@@ -0,0 +1,90 @@
+Django-locking
+==============
+
+About
+-----
+
+`django-locking` provides a system that makes concurrent editing impossible, and informs users of what other users are working on and for how long that content will remain locked. Users can still read locked content, but cannot modify or save it.
+
+Requirements
+-------------
+
+* Django 2.2
+* Python 3.6 - 3.8
+
+Installation
+------------
+
+1. Download and install django-locking.
+
+    ```console
+    $ pip install git+git://github.com/unt-libraries/django-locking.git
+    ```
+
+2. Add app to INSTALLED_APPS.
+
+    ```python
+    INSTALLED_APPS = (
+        'locking',
+    )
+    ```
+
+Developing
+----------
+
+1. Clone the git repository:
+
+    ```python
+        git clone https://github.com/unt-libraries/django-locking.git
+    ```
+
+2. Navigate into the cloned repository:
+
+    ```python
+        cd django-locking
+    ```
+
+3. Install the requirements (preferably in a virtual environment):
+
+    ```python
+        python setup.py install
+    ```
+
+4. Run the migrations:
+
+    ```python
+        python manage.py migrate
+    ```
+
+5. Create a superuser so you can log into the admin interface
+
+    ```python
+        python manage.py createsuperuser
+    ```
+
+6. Start the test server:
+
+    ```python
+        python manage.py runserver
+    ```
+
```

Comment:
Should we add a step saying where to view the interface we are running?

Reply:
yeah...I will add it here.